repo2docker/buildpacks/base.py

```diff
@@ -608,7 +609,33 @@ def _filter_tar(tar):
         for fname in ("repo2docker-entrypoint", "python3-login"):
             tar.add(os.path.join(HERE, fname), fname, filter=_filter_tar)
 
-        tar.add(".", "src/", filter=_filter_tar)
+        exclude = []
+
+        for ignore_file_name in [".dockerignore", ".containerignore"]:
+            if os.path.exists(ignore_file_name):
```

Comment:
```suggestion 
            ignore_file_name = self.binder_path(ignore_file_name)
            if os.path.exists(ignore_file_name):
```

and it's worth a test to show that top-level .dockerignore is ignored if there's a `binder` directory. This can be added to `tests/venv/binder-dir`

Reply:
Done